.github/workflows/BuildImage.yml

```diff
@@ -3,26 +3,32 @@ name: Build Image
 on: [push, pull_request, workflow_dispatch]
 
 env:
-  ENDPOINT: "linuxserver/mods" #don't modify
-  BASEIMAGE: "replace_baseimage" #replace
-  MODNAME: "replace_modname" #replace
+  ENDPOINT: "linuxserver/mods"
+  BASEIMAGE: "transmission"
+  MODNAME: "floodui"
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2.3.3
+      - uses: actions/checkout@v3.1.0
 
       - name: Build image
         run: |
+          FLOOD_VERSION=$(curl -s https://api.github.com/repos/johman10/flood-for-transmission/releases/latest | jq -rc ".tag_name")
+          echo "FLOOD_VERSION=${FLOOD_VERSION}" >> $GITHUB_ENV
           docker build --no-cache -t ${{ github.sha }} .
```

Comment:
If you inject the version as build arg here, we can make sure the tags are accurate

Reply:
Fixed in 1bbad6e